plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java

```diff
@@ -69,10 +69,10 @@ public JdbcMetadataSessionProperties(JdbcMetadataConfig jdbcMetadataConfig, @Max
                         jdbcMetadataConfig.isTopNPushdownEnabled(),
                         false))
                 .add(integerProperty(
-                        INSERT_BATCH_SIZE,
-                        "Insert batch size",
-                        jdbcMetadataConfig.getInsertBatchSize(),
-                        value -> validateInsertBatchSize(value, MAX_ALLOWED_INSERT_BATCH_SIZE),
+                        WRITE_BATCH_SIZE,
+                        "Write batch size",
```

Comment:
Synchronize this with `@ConfigDescription`

Reply:
done